examples/ICESat-2_DAAC_DataAccess_Example.ipynb

```diff
@@ -2,7 +2,6 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.                start_time='03:30:00', end_time='21:30:00', version='004')
</pre><p>The latest icesat2 data is version 004, if set the version 002, the error occurs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes - that is the intended behavior, as per the text block above:
>NOTE Version 002 is used as an example in the below cell. However, using it will cause 'no results' errors in granule ordering for some search parameters. These issues have been resolved in later versions of the data products, so it is best to use the most recent version where possible. Thus, you will need to change the version associated with region_a and rerun the next cell for the rest of this notebook to run.